common/src/Module.php

```diff
@@ -422,6 +450,22 @@ private function sendInvites($ps_shop_id) {
         }
     }
 
+    private function getParsedOrderLines($order_lines) {
```

Comment:
I would suggest renaming this to `getProducts`
```suggestion
    private function getProducts($order_lines) {
```